### PR TITLE
Add brew switch to README

### PR DESCRIPTION
**Pull request by ogom**
"Can't open shared library" ERROR in plugin installation.
So I'm added brew switch to README.

```
mysql> INSTALL PLUGIN Mroonga SONAME 'ha_mroonga.so';
ERROR 1126 (HY000): Can't open shared library '/usr/local/Cellar/mysql56/5.6.34/lib/plugin/ha_mroonga.so' (errno: 2 dlopen(/usr/local/Cellar/mysql56/5.6.34/lib/plugin/ha_mroonga.so, 2): image not found)
```

## process of brew switch 

* Install mysql of old version

```
$ brew install homebrew/versions/mysql56
==> Installing mysql56 from homebrew/versions
==> Downloading https://homebrew.bintray.com/bottles-versions/mysql56-5.6.34.el_capitan.bottle.tar.gz
$ mysql --version
mysql  Ver 14.14 Distrib 5.6.34, for osx10.11 (x86_64) using  EditLine wrapper
$ mysql -uroot
mysql> show variables like 'version';
+---------------+--------+
| Variable_name | Value  |
+---------------+--------+
| version       | 5.6.34 |
+---------------+--------+
```

* stop mysql

```
$ launchctl unload ~/Library/LaunchAgents/homebrew.mxcl.mysql56.plist
$ mysql.server stop
```

* Install mroonga

```
$ brew install https://raw.github.com/mroonga/homebrew/master/mroonga.rb --use-homebrew-mysql
==> Downloading http://packages.groonga.org/source/mroonga/mroonga-6.10.tar.gz
Already downloaded: /Users/ogom/Library/Caches/Homebrew/mroonga-6.10.tar.gz
==> Downloading https://cdn.mysql.com/Downloads/MySQL-5.7/mysql-boost-5.7.16.tar.gz
Already downloaded: /Users/ogom/Library/Caches/Homebrew/mysql-5.7.16.tar.gz
```

* brew switch  mysql of old version

```
$ brew switch mysql 5.7.16
$ brew link --overwrite mysql
$ mysql --version
mysql  Ver 14.14 Distrib 5.7.16, for osx10.11 (x86_64) using  EditLine wrapper
$ mysql_upgrade -u root
$ mysql.server start
$ mysql -uroot
mysql> show variables like 'version';
+---------------+--------+
| Variable_name | Value  |
+---------------+--------+
| version       | 5.7.16 |
+---------------+--------+

mysql> INSTALL PLUGIN Mroonga SONAME 'ha_mroonga.so';
mysql> show variables like 'mroonga_version';
+-----------------+-------+
| Variable_name   | Value |
+-----------------+-------+
| mroonga_version | 6.10  |
+-----------------+-------+
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ With MariaDB:
     % brew tap mroonga/mroonga
     % brew install mroonga --use-homebrew-mariadb
 
+If you have an old mysql, please switch to the latest version:
+
+    % mysql.server stop
```
Comment (cosmo0920) [inline README.md]: Could you use `brew services stop mysql` here?
`brew services` sub command is more modern way to stop macOS services than this one.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ With MariaDB:
...
+    % brew switch mysql 5.7.16
+    % brew link --overwrite mysql
+    % mysql.server start
```
Comment (cosmo0920) [inline README.md]: Could you use `brew services start mysql` here?

File: README.md [hunk 3]
```diff
@@ -10,6 +10,13 @@ With MariaDB:
     % brew tap mroonga/mroonga
     % brew install mroonga --use-homebrew-mariadb
 
+If you have an old mysql, please switch to the latest version:
+
+    % mysql.server stop
+    % brew switch mysql 5.7.16
+    % brew link --overwrite mysql
```
Comment (cosmo0920) [inline README.md]: `brew link --overwrite <Some Formula name>` is dangerous operation.
Is there any safe way to achieve this?

**Comment by ogom** [comment]
It is a new process to unlink before installing.

## process of brew unlink

* Install mysql of old version

```
$ brew install homebrew/versions/mysql56
$ brew services start mysql56
$ mysql -uroot
mysql> show variables like 'version';
```

* stop mysql

```
$ brew services stop mysql56
$ brew unlink mysql56
$ mv /usr/local/var/mysql /usr/local/var/mysql_56
```

* Install mroonga

```
$ brew install https://raw.github.com/mroonga/homebrew/master/mroonga.rb --use-homebrew-mysql
$ brew services start mysql
$ mysql -uroot < '/usr/local/Cellar/mroonga/6.10/share/mroonga/install.sql'
```

File: README.md (hunk 3 above)
Comment (ogom) [inline README.md]: How about brew unlink?

File: README.md (hunk 1 above)
Comment (ogom) [inline README.md]: Use brew services stop mysql

File: README.md (hunk 2 above)
Comment (ogom) [inline README.md]: Use brew services start mysql

File: README.md [hunk 4]
```diff
@@ -10,12 +10,11 @@ With MariaDB:
...
-If you have an old mysql, please switch to the latest version:
+If you have an old formulae, please unlink to the old formulae version:
 
-    % mysql.server stop
-    % brew switch mysql 5.7.16
-    % brew link --overwrite mysql
-    % mysql.server start
+    % brew services stop [formulae]
+    % brew unlink [formulae]
+    % mv /usr/local/var/[formulae] /usr/local/var/[formulae_version]
```
Comment (cosmo0920) [inline README.md]: Can not `brew link [new formula]` achieve this?

Comment (ogom) [inline README.md]: If it unlinks first, the mysql command becomes the latest version on installation of mroonga.

Comment (cosmo0920) [inline README.md]: Ah, I got it.
You mean that this command is intended to back up old mysql data?
If so, it is reasonable for me.

**Pull request merged**